### PR TITLE
Uat into prod

### PR DESCRIPTION
**Pull request by Chee-Ng**
- Added Pardot tracking code and the ability to send CCS contact form data to Pardot Form Handler in Pardot
- Added redirection /workforce to /products-and-services/people/workforce/
- Fixed 500 error when user trying to search for a agreement or supplier
- More checking for GM banner and the landing LandingPage for FaT
- Allow CMS user to choose if the option card should show or not on pages
- Allow CMS user to customise: Homepage Components, Option Cards and Upcoming Deals Page
- Form Validation for all newsletter form

### CODE REVIEW

**Review by zubairali6 — APPROVED** [review]

File: config/categories.yaml [hunk 1]
```diff
@@ -90,3 +90,12 @@ pillars:
...
         slug: 'technology-products-services'
+
+  - name: 'General Common Goods and Services'
+    db_value: 'General Common Goods and Services'
+    slug: 'general-common-goods-and-services'
+
+    categories:
+      - name: 'Below Threshold'
+        db_value: 'Below Threshold'
+        slug: 'below-threshold'
```
Comment (karishma-ccs) [inline config/categories.yaml:101]: Where has this come from?

**Review by karishma-ccs — CHANGES_REQUESTED** [review]
Just noticed this category in UAT - doesn't seem right?

File: config/categories.yaml (hunk 1 above)
Comment (Chee-Ng) [reply config/categories.yaml:101]: https://crowncommercialservice.atlassian.net/browse/WEB-1448

Comment (karishma-ccs) [reply config/categories.yaml:101]: Should it show up in uat/prod if there's no frameworks associated with it? Let's get clarification from Nicola, as in the ticket she seems to suggest it's backend work that won't affect the users

Comment (Chee-Ng) [reply config/categories.yaml:101]: I will mention this on the group chat

Comment (Chee-Ng) [reply config/categories.yaml]: removed from this deployment

**Pull request merged**